docs/maui/views/Popup.md

```diff
@@ -188,23 +197,24 @@ The `DefaultPopupSettings` provided will be applied as the default value for eve
 ```cs
 .UseMauiCommunityToolkit(static options =>
 {
-  options.SetPopupDefaults(new DefaultPopupSettings
-  {
-	  CanBeDismissedByTappingOutsideOfPopup = true,
-	  BackgroundColor = Colors.Orange,
-	  HorizontalOptions = LayoutOptions.End,
-	  VerticalOptions = LayoutOptions.Start,
-	  Margin = 72,
-	  Padding = 4
-  });
-  options.SetPopupOptionsDefaults(new DefaultPopupOptionsSettings
-  {
-	  CanBeDismissedByTappingOutsideOfPopup = true,
-	  OnTappingOutsideOfPopup = async () => await Toast.Make("Popup Dismissed").Show(CancellationToken.None),
-	  PageOverlayColor = Colors.Orange,
-	  Shadow = null,
-	  Shape = null
-  });
+    options.SetPopupDefaults(new DefaultPopupSettings
+    {
+        CanBeDismissedByTappingOutsideOfPopup = true,
+        BackgroundColor = Colors.Orange,
+        HorizontalOptions = LayoutOptions.End,
+        VerticalOptions = LayoutOptions.Start,
+        Margin = 72,
+        Padding = 4
+    });
+
+    options.SetPopupOptionsDefaults(new DefaultPopupOptionsSettings
+    {
+        CanBeDismissedByTappingOutsideOfPopup = true,
+        OnTappingOutsideOfPopup = async () => await Toast.Make("Popup Dismissed").Show(CancellationToken.None),
+        PageOverlayColor = Colors.Orange,
+        Shadow = null,
+        Shape = null
+    ß});
```

Comment:
This beta symbol doesn't look as intended

Reply:
Great spot thanks!